fdbserver/RESTKmsConnector.actor.cpp

```diff
@@ -1025,15 +1053,15 @@ ACTOR Future<Void> procureValidationTokensFromFiles(Reference<RESTKmsConnectorCt
 
 		tokensPayloadSize += fSize;
 		if (tokensPayloadSize > SERVER_KNOBS->REST_KMS_CONNECTOR_VALIDATION_TOKENS_MAX_PAYLOAD_SIZE) {
-			TraceEvent("ValidationTokenPayloadTooLarge", ctx->uid)
+			TraceEvent(SevWarnAlways, "RESTValidationTokenPayloadTooLarge", ctx->uid)
 			    .detail("MaxAllowedSize", SERVER_KNOBS->REST_KMS_CONNECTOR_VALIDATION_TOKENS_MAX_PAYLOAD_SIZE);
 			throw value_too_large();
 		}
 
 		state Standalone<StringRef> buff = makeString(fSize);
 		int bytesRead = wait(tFile->read(mutateString(buff), fSize, 0));
 		if (bytesRead != fSize) {
-			TraceEvent("DiscoveryKmsUrlFileReadShort", ctx->uid)
+			TraceEvent(SevError, "RESTDiscoveryKmsUrlFileReadShort", ctx->uid)
```

Comment:
Why is this the only one that logs at `SevError` level?

Reply:
Short read are `filesystem error`, hence, SevError seems like the right thing here.